app/renderer/js/pages/preference/general-section.js

```diff
@@ -120,6 +132,22 @@ class GeneralSection extends BaseComponent {
 		});
 	}
 
+	initSidebarToggle() {
+		this.$sidebarOptionSettings = document.querySelector('#ui-option-settings #sidebar-option .setting-control');
+		this.$sidebarOptionSettings.innerHTML = '';
+
+		const sidebarOption = ConfigUtil.getConfigItem('show-sidebar', false);
```

Comment:
It would be better if we show sidebar initially. If we hide it on the first load, users might not able to find it. What do you think?